package.json

```diff
@@ -132,5 +132,6 @@
   },
   "resolutions": {
     "electron-builder": "^23.0.0"
-  }
+  },
+  "packageManager": "yarn@1.22.19"
```

Comment:
Why lock this version of yarn?

Reply:
I don't know 😱😱😱
I tried `yarn` version 3 in another project, but it was worst, and I tried to force using `yarn classic` but it returned back to version 3 every time, until I declared it here!